src/Morphic-Base.package/PragmaMenuBuilder.extension/instance/fallbackMenu.st

```diff
@@ -0,0 +1,4 @@
+*Morphic-Base
+fallbackMenu
+
+	^ FallbackMenu when: self fails: model.
```

Comment:
The FallbackMenu is a Morphic specific behavior dependent on debugger and Workspace